src/bcrhp/bcrhp/migrations/909_update_url_prefix.py

```diff
@@ -0,0 +1,542 @@
+                )
+            )
+            where tiledata -> 'dc12f414-0d07-11ed-8804-5254008afee6' is not null
+            and (tiledata -> 'dc12f414-0d07-11ed-8804-5254008afee6') -> 0 -> 'content' is not null
+            ;
+            -- for site images
+            update tiles
+            set tiledata = jsonb_set(
+                tiledata,
+                '{0a883b80-2fb6-11ed-be5f-5254008afee6}',
+                jsonb_set(
+                    (tiledata -> '0a883b80-2fb6-11ed-be5f-5254008afee6')::jsonb,
+                    '{0, url}',
+                    ('"' || replace((tiledata -> '0a883b80-2fb6-11ed-be5f-5254008afee6') -> 0 ->> 'url', '/bcrhp/', '/int/bcrhp/') || '"')::jsonb
+                )
+            )
+            where tiledata -> '0a883b80-2fb6-11ed-be5f-5254008afee6' is not null
+            and (tiledata -> '0a883b80-2fb6-11ed-be5f-5254008afee6') -> 0 -> 'url' is not null
+            ;
+            update tiles
+            set tiledata = jsonb_set(
+                tiledata,
+                '{0a883b80-2fb6-11ed-be5f-5254008afee6}',
+                jsonb_set(
+                    (tiledata -> '0a883b80-2fb6-11ed-be5f-5254008afee6')::jsonb,
+                    '{0, content}',
+                    ('"' || replace((tiledata -> '0a883b80-2fb6-11ed-be5f-5254008afee6') -> 0 ->> 'content', '/bcrhp/', '/int/bcrhp/') || '"')::jsonb
+                )
+            )
+            where tiledata -> '0a883b80-2fb6-11ed-be5f-5254008afee6' is not null
+            and (tiledata -> '0a883b80-2fb6-11ed-be5f-5254008afee6') -> 0 -> 'content' is not null
+            ;
+            commit;
+            """
+        ),
+        RunPrivilegedSQL(
+            """
+            begin;
+            update nodes
+            set config = jsonb_set(
+                config,
+                '{searchString}',
+                ('"' || replace(config->>'searchString', '/int/bcrhp/', '/bcrhp/') || '"')::jsonb
+            )
+            where (config->'searchString') is not null
+            ;
+            commit;
+            """,
+            """
+            begin;
+            update nodes
+            set config = jsonb_set(
+                config,
+                '{searchString}',
+                ('"' || replace(config->>'searchString', '/bcrhp/', '/int/bcrhp/') || '"')::jsonb
+            )
+            where (config->'searchString') is not null
+            ;
+            commit;
+            """
+        ),
+        RunPrivilegedSQL(
+            """
+            begin;
+            update concepts
+            set legacyoid = replace(legacyoid, '/int/bcrhp/', '/bcrhp/')
+            ;
+            commit;
+            """,
+            """
+            begin;
+            update concepts
+            set legacyoid = replace(legacyoid, '/bcrhp/', '/int/bcrhp/')
+            ;
+            commit;
+            """
+        ),
+        RunPrivilegedSQL(
+            """
+            begin;
+            update values
```

Comment:
As above - please limit all updates to rows that should be updated.

Reply:
Done